$OpenBSD: patch-src_utils_DivModBySame_hh,v 1.2 2017/08/03 08:35:06 bentley Exp $

Disable arm assembly code that asks for too many registers.

--- src/utils/DivModBySame.hh.orig	Sun May  7 00:49:55 2017
+++ src/utils/DivModBySame.hh	Sun May  7 00:50:01 2017
@@ -156,7 +156,7 @@ class DivModBySame (public)
 		);
 		return _cl_;
 	#endif
-	#elif defined(__arm__)
+	#elif defined(notdef)
 		uint32_t res;
 		uint32_t dummy;
 		asm volatile (
@@ -217,7 +217,7 @@ class DivModBySame (public)
 
 	uint32_t mod(uint64_t dividend) const
 	{
-	#ifdef __arm__
+	#ifdef notdef
 		uint32_t res;
 		uint32_t dummy;
 		asm volatile (
